dev_envs/mac-env-setup.md

```diff
@@ -106,6 +109,128 @@ After installing Brew, you'll want to install other useful
 packages. We recommend installing all the packages specified in the
 [CISA `laptop` script repository](https://github.com/cisagov/laptop/blob/master/Brewfile).
 
+### Colima: A Docker alternative ###
+
+For individuals or businesses seeking a container solution without
+the licensing conditions associated with [Docker Desktop](https://www.docker.com/pricing/faq/)
+, [Colima](https://github.com/abiosoft/colima) offers a compelling
+open-source alternative. Compatible with both MacOS and Linux,
+Colima facilitates running containers directly using Docker
+commands without the need for Docker Desktop.
+
+#### Installing Colima ####
+
+**Prerequisites:** If Docker Desktop is installed on your system,
+uninstall it before proceeding.
+
+**Installation Steps:**
+
+1. Install Colima and Docker command-line tools using Homebrew:
+
+    ```console
+    brew install colima docker docker-compose
+    ```
+
+1. Verify the installation by starting Colima:
+
+    ```console
+    colima start
+    ```
+
+    This initializes the Docker daemon within a lightweight VM.
+    The first startup may take a moment.
+
+1. Confirm Docker is operational:
+
+    ```console
+    docker
+    ```
+
+    You should see the Docker usage menu.
+
+1. Verify `docker-compose` is working:
+
+    ```console
+    docker-compose
+    ```
+
+    The `docker-compose` usage menu should appear.
+
+### Optional configurations ###
+
+Enhance your Colima setup with the following optional configurations:
+
+#### Docker Compose as a Docker plugin ####
```

Comment:
So the `docker-compose` formula _is_ the Compose plugin. You can see in the [formula's code](https://github.com/Homebrew/homebrew-core/blob/277c6de80b2e5ba38b11bf0fff467c1ad8b2fdb3/Formula/d/docker-compose.rb) that it's pulling down the latest version. It just creates a `docker-compose` binary that is symlinked into the plugins directory.

Reply:
Good catch! Verified it on my end. Everything looks good without the config stuff.

See here: https://github.com/cisagov/development-guide/pull/96/commits/8ce2503e2986f93318c97fd1ea32a649917c989f